src/ic-ledger-types/src/lib.rs

```diff
@@ -298,6 +298,27 @@ pub async fn transfer(
     Ok(result)
 }
 
+#[derive(Serialize, Deserialize, CandidType, Clone, Hash, Debug, PartialEq, Eq)]
+pub struct Symbol {
+    pub symbol: String,
+}
+
+/// Calls the "token_symbol" method on the specified canister.
+/// # Example
+/// ```no_run
+/// use candid::Principal;
+/// use ic_cdk::api::{caller, call::call};
+/// use ic_ledger_types::{Symbol, token_symbol};
+///
+/// async fn symbol(ledger_canister_id: Principal) -> String {
+///   token_symbol(ledger_canister_id).await.expect("call to ledger failed").symbol
+/// }
+/// ```
```

Comment:
Not sure it's a good example, it doesn't convey any new information in addition to the type signature. Might be worth leaving just the summary doc string.